main.py

```diff
@@ -486,10 +465,10 @@ def prep_IR(ir: str) -> str:
 
     # Compile
     if is_gcc:
-        case.bad_setting.add_flag("-emit-llvm")
-        good_setting.add_flag("-emit-llvm")
-        asm_bad = utils.get_asm_str(source, case.bad_setting, bldr)
-        asm_good = utils.get_asm_str(source, good_setting, bldr)
+        case.bad_setting.flags.append("-emit-llvm")
+        good_setting.flags.append("-emit-llvm")
```

Comment:
As `subprocess` is a bit finicky with flags I suggest to implement getter and setter methods for them.

Reply:
I was blindly converting the old code, but now I realize that this is guarded by `is_gcc` so we shouldn't be using `-emit-llvm`, this should be wrong right?

https://github.com/DeadCodeProductions/dead/blob/main/main.py#L489

Reply:
Also, what would these getters/settes be doing? Some kind of sanitization?

Reply:
this seems very wrong.

The setters should basically do a `.strip().split(' ')` as spaces trip up `subprocess.run`. 
It could also handle the case when you have no flags set yet. For example, if I do something like 
```python
f"clang -O3 code.c {' '.join(setting.flags)}"
```
it breaks when I have no flags. 
I think `run_cmd` does sanitize commands a bit (`filter(bool, cmd_str.split(' '))`). 

Also for saving and comparing the flags in a database, such cleanup is important.